geopandas/tests/test_geom_methods.py

```diff
@@ -1466,3 +1466,36 @@ def test_get_coordinates_not(self):
             NotImplementedError, match="shapely >= 2.0 or PyGEOS are required"
         ):
             self.g11.get_coordinates()
+
+    def test_minimum_bounding_radius(self):
```

Comment:
```suggestion
    @pytest.mark.skipif(
        not (compat.USE_PYGEOS or compat.USE_SHAPELY_20),
        reason="minimum_bounding_radius not implemented for shapely<2",
    )
    def test_minimum_bounding_radius(self):
```

You need to skip this test for CI environments that still use shapely 1.8 (for backwards compatibility). And as the `test_get_coordinates_not`, you should write a tiny another one flipping the condition and checking the error.

Reply:
added!